app/javascript/app/pages/ndc-country/ndc-country-component.jsx

```diff
@@ -13,26 +15,117 @@ import Dropdown from 'components/dropdown';
 import { NDC_COUNTRY } from 'data/SEO';
 import { MetaDescription, SocialMetadata } from 'components/seo';
 import { TabletLandscape } from 'components/responsive';
+import longArrowBack from 'assets/icons/long-arrow-back.svg';
+import { toStartCase } from 'app/utils';
 
 import anchorNavRegularTheme from 'styles/themes/anchor-nav/anchor-nav-regular.scss';
 import theme from 'styles/themes/dropdown/dropdown-links.scss';
 import lightSearch from 'styles/themes/search/search-light.scss';
 import styles from './ndc-country-styles.scss';
 
+const FEATURE_LTS_EXPLORE = process.env.FEATURE_LTS_EXPLORE === 'true';
+
+const getPreviousPathLabel = previousPathname => {
+  const updatedPathname = previousPathname;
+  let lastPathLabel = {
+    '/': 'Home'
+  }[previousPathname];
+  const regexs = [
+    { regex: /countries\/compare/, label: 'country compare' },
+    { regex: /countries/, label: 'country' }
+  ];
+
+  regexs.some(regexWithLabel => {
+    const { regex, label } = regexWithLabel;
+    if (previousPathname && previousPathname.match(regex)) {
+      lastPathLabel = label;
+      return true;
+    }
+    return false;
+  });
+  return lastPathLabel || (updatedPathname && toStartCase(updatedPathname));
+};
+
+const shouldClearPath = pathname => {
+  const clearRegexps = [/\/ndcs\/country/, /\/error-page/, /\/ndcs\/compare/];
+  if (clearRegexps.some(r => pathname.match(r))) {
```

Comment:
This bit throws an error after copying & pasting the country link:
![ah23u-x5xqh](https://user-images.githubusercontent.com/15097138/69956291-7c1d6400-14f7-11ea-83d1-84aca6d109aa.gif)
![image](https://user-images.githubusercontent.com/15097138/69956000-d23dd780-14f6-11ea-857c-c3d2aa50578a.png)